### PR TITLE
OA-7: Setup tests for omod, test MetadataController

### PR DESCRIPTION
**Pull request by pjsier**
This PR sets up some basic tests for the `omod` layer on the `MetadataController` to implement OA-7: https://issues.openmrs.org/browse/OA-7

I wasn't sure how far to go in testing the values inside the controller, and I'm happy to build those out more if you're looking for more than regression tests. I don't normally work in Java, and I think I have my editor setup with the style configuration, but if I'm doing not doing anything the way it should be done let me know, thanks!

### CODE REVIEW

**Comment by Devanshk9** [comment]
Hi @pjsier  please resolve merge conflicts

**Comment by pjsier** [comment]
@Devanshk9 just updated, let me know if it needs anything else, thanks!

**Comment by Devanshk9** [comment]
Ok now merge conflict it resolved😊
thanks

**Comment by Devanshk9** [comment]
Please also squash your commits once there are no changes required.

**Comment by pjsier** [comment]
@Devanshk9 rebased now so it should be fixed, thanks!

File: omod/src/test/java/org/openmrs/module/oauth2/web/controller/MetadataControllerTest.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+import org.springframework.http.ResponseEntity;
+
+public class MetadataControllerTest extends BaseModuleWebContextSensitiveTest {
+	
+	@Autowired
+	private MetadataController controller;
+	
+	@Test
+	public void shouldNotBeNull() {
+		assertNotNull(controller);
```
Comment (ibacher) [inline omod/src/test/java/org/openmrs/module/oauth2/web/controller/MetadataControllerTest.java]: I'd eliminate this test, which is really just verifying that Spring works and not a useful test of any module-specific functionality.

**Review by ibacher — CHANGES_REQUESTED** [review]
@pjsier This looks useful. Could you get rid of the one one test? Then I'll be happy with it.

**Comment by pjsier** [comment]
@ibacher just updated, thanks for the review!

**Review by ibacher — APPROVED** [review]

**Pull request merged**